services/webnn/ort/graph_builder_ort.cc

```diff
@@ -1532,6 +1533,40 @@ void GraphBuilderOrt::AddTransposeOperation(const mojom::Transpose& transpose) {
                          attributes);
 }
 
+void GraphBuilderOrt::AddSplitOperation(const mojom::Split& split) {
+  const std::string node_name = GenerateNextOperationName(split.label);
+  const std::string input_name = GetOperandNameById(split.input_operand_id);
+  std::vector<const char*> input_names = {input_name.c_str()};
+
+  const auto output_nums = split.output_operand_ids.size();
+  base::FixedArray<int64_t> split_sizes(output_nums);
+  for (size_t i = 0; i < output_nums; i++) {
+    const std::vector<uint32_t>& output_shape =
+        GetOperand(split.output_operand_ids[i]).descriptor.shape();
+    CHECK_LT(split.axis, output_shape.size());
+    split_sizes[i] = base::checked_cast<int64_t>(output_shape[split.axis]);
+  }
+  const std::string split_name = CreateInitializer<int64_t>(
```

Comment:
Please add a comment explaining the usage of `split` input and when it is used the attribute `num_outputs` is not specified.

Reply:
Added more comments.